README.md

```diff
@@ -0,0 +1,19 @@
+# [itch](https://github.com/itchio/itch)
+
+Install, update and play indie games
+
+**NOTICE: This package is not verified by, affiliated with, or supported by itch.io.**
+
+## Notes
+
+Uses wine as a base, to provide support for windows games.
```

Comment:
Please note that not building from source and using Wine as base app instead of Electron is controversial. https://github.com/itchio/itch/issues/2793

Reply:
This is not controversial, the sayings of one person does not make it controversial.